frontend/simple/sass/_groupincome.scss

```diff
@@ -74,3 +74,22 @@
   background-color: #1fc8db;
   border-radius: 20px;
 }
+
+//--------------------------------------
+// Modifiers
+//--------------------------------------
+.is-danger {
+  color: $red;
+}
+.is-success {
+  color: $green;
+}
+.is-warning {
+  color: $orange;
+}
+.is-light {
+  color: $grey;
+}
+.is-small {
+  font-size: $size-7;
+}
```

Comment:
These look like they're just copied from Bulma? Why are they necessary here?

If they are necessary for some reason, can you add that explanation in a comment to the source as well?